include/zephyr/drivers/video-controls.h

```diff
@@ -1,85 +1,168 @@
-/**
- * @file
- *
- * @brief Public APIs for Video.
- */
-
 /*
  * Copyright (c) 2019 Linaro Limited.
+ * Copyright (c) 2024 tinyVision.ai Inc.
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 #ifndef ZEPHYR_INCLUDE_VIDEO_CONTROLS_H_
 #define ZEPHYR_INCLUDE_VIDEO_CONTROLS_H_
 
+/**
+ * @file
+ *
+ * @brief Public APIs for Video.
+ */
+
 /**
  * @brief Video controls
  * @defgroup video_controls Video Controls
  * @ingroup io_interfaces
+ *
+ * The Video control IDs (CIDs) are introduced with the same name as
+ * Linux V4L2 subsystem and under the same class. This facilitates
+ * inter-operability and debugging devices end-to-end across Linux and
+ * Zephyr.
+ *
+ * This list is maintained compatible to the Linux kernel definitions in
+ * @c linux/include/uapi/linux/v4l2-controls.h
+ *
  * @{
  */
 
-#include <zephyr/device.h>
 #include <stddef.h>
-#include <zephyr/kernel.h>
 
+#include <zephyr/device.h>
```

Comment:
The `video-controls.h` only defines CID macros. It uses nothing from `device.h`, `kernel.h`, etc. Although it's not in this PR scope but I recommend to drop all included headers.

Reply:
Good point! Instead of shuffling these headers, I removed them.
This might have been code added and removed later (refactoring).